sdk/servicebus/service-bus/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     /* Basic Options */
     "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
     "module": "es6" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    "lib": [] /* lib dependencies are triple-slash directives in lib/index.ts */,
+    "lib": ["esnext"] /* lib dependencies are triple-slash directives in lib/index.ts */,
```

Comment:
Remove this lib entry and fix up the triple-slash references above and everything should work fine!

Reply:
fixed